### ISSUE TITLE
docker deploying to DigitalOcean App returns this error in log at the end - cannot find module './lib/pdf'

### ISSUE THREAD

**Issue by anjanesh** [typpo/quickchart#113]
```
[quickchart] [2021-11-27 06:49:59] => Uploaded the built image to the container registry
[quickchart] [2021-11-27 06:49:59] => Build complete
[quickchart] [2021-11-27 06:50:54] {"name":"quickchart","hostname":"quickchart-77xxx-xxx","pid":1,"level":30,"msg":"Telemetry is enabled","time":"2021-11-27T06:50:54.083Z","v":0}
[quickchart] [2021-11-27 06:50:54] internal/modules/cjs/loader.js:818
[quickchart] [2021-11-27 06:50:54]   throw err;
[quickchart] [2021-11-27 06:50:54]   ^
[quickchart] [2021-11-27 06:50:54]
[quickchart] [2021-11-27 06:50:54] Error: Cannot find module './lib/pdf'
[quickchart] [2021-11-27 06:50:54] Require stack:
[quickchart] [2021-11-27 06:50:54] - /quickchart/index.js
[quickchart] [2021-11-27 06:50:54]     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
[quickchart] [2021-11-27 06:50:54]     at Function.Module._load (internal/modules/cjs/loader.js:667:27)
[quickchart] [2021-11-27 06:50:54]     at Module.require (internal/modules/cjs/loader.js:887:19)
[quickchart] [2021-11-27 06:50:54]     at require (internal/modules/cjs/helpers.js:74:18)
[quickchart] [2021-11-27 06:50:54]     at Object.<anonymous> (/quickchart/index.js:11:55)
[quickchart] [2021-11-27 06:50:54]     at Module._compile (internal/modules/cjs/loader.js:999:30)
[quickchart] [2021-11-27 06:50:54]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
[quickchart] [2021-11-27 06:50:54]     at Module.load (internal/modules/cjs/loader.js:863:32)
[quickchart] [2021-11-27 06:50:54]     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
[quickchart] [2021-11-27 06:50:54]     at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12) {
[quickchart] [2021-11-27 06:50:54]   code: 'MODULE_NOT_FOUND',
[quickchart] [2021-11-27 06:50:54]   requireStack: [ '/quickchart/index.js' ]
[quickchart] [2021-11-27 06:50:54] }
```

**Comment by typpo** [comment]
Hi Anjanesh, I can't test on DO App, but I'm guessing the issue has to do with how you're supplying the files to be built, especially because it says `lib/pdf.js` is missing.

The docker image builds and runs just fine locally, with `docker build -t ianw/quickchart . && docker run -it -p 3400:3400 ianw/quickchart`

**Comment by anjanesh** [comment]
Oh, so this is not a regular DO droplet which is a VPS. DO has something known as **App** where one can connect to a github / gitlab / bitbucket repo and the DO app will auto run from it after building it - either it'll detect what kind of app it is or, if it finds a Dockerfile then it'll use docker. So I just cloned quickchart at https://github.com/anjanesh/quickchart and connected my DO App to it'll take it from there. But the build failed at the end.

I did a docker pull ianw/quickchart on my local Ubuntu system and it worked, so I don't see any issue with the Dockerfile.
